docker/test/unit/process_unit_tests_result.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+import os
+import logging
+import argparse
+import csv
+
+OK_SIGN = 'OK ]'
+FAILED_SIGN = 'FAILED  ]'
```

Comment:
Double whitespace.

Reply:
As it should be. Output of gtest looks like this:
```
[----------] 4 tests from ColumnUnique
[ RUN      ] ColumnUnique.InsertRange
[       OK ] ColumnUnique.InsertRange (5810 ms)
[ RUN      ] ColumnUnique.InsertRangeWithOverflow
../src/Columns/tests/gtest_column_unique.cpp:83: Failure
Expected equality of these values:
  idx->size()
    Which is: 1000000
  num_values + 1
    Which is: 1000001
[  FAILED  ] ColumnUnique.InsertRangeWithOverflow (5937 ms)
```